ansible/roles/jonatasbaldin.zabbix3agent/.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
i added this to our repo via ansible galaxy. is there a way to keep it out of the repo entirely? I also needed to modify the role slightly to set certain variables